### PR TITLE
Implement header navigation

### PR DESCRIPTION
**Pull request by TN4HPEL3**

### CODE REVIEW

File: src/components/organisms/NavigationListOrganisms/NavigationListOrganisms.jsx
Comment (AnnaIstochnikova) [inline src/components/organisms/NavigationListOrganisms/NavigationListOrganisms.jsx:1]: Shouldn't it be the <nav> tag instead of the <ul>?

Comment (AnnaIstochnikova) [inline src/components/organisms/NavigationListOrganisms/NavigationListOrganisms.jsx:1]: Shouldn't it be the nav tag instead of the ul?

**Review by AnnaIstochnikova — CHANGES_REQUESTED** [review]

**Review by AnnaIstochnikova — APPROVED** [review]

**Review by AnnaIstochnikova — CHANGES_REQUESTED** [review]
Are you sure we want to merge this branch into 'main' instead of 'dev'?

**Comment by TN4HPEL3** [comment]
> Are you sure we want to merge this branch into 'main' instead of 'dev'?

😅

**Comment by TN4HPEL3** [comment]
Conflicts resolved. @AnnaIstochnikova

**Review by AnnaIstochnikova — APPROVED** [review]

**Pull request merged**